examples/gpt/single/run_eval.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import math
+import time
+import sys
+import random
+import numpy as np
+
+import paddle
+from paddle.io import DataLoader
+
+sys.path.append("../../../")
```

Comment:
We may use relative import:
https://peps.python.org/pep-0328/#guido-s-decision

<img width="807" alt="image" src="https://user-images.githubusercontent.com/10721757/186846685-ea91a255-0c3a-4554-9277-0f1ff7ad7306.png">